gcp_deepvariant_runner.py

```diff
@@ -210,17 +195,72 @@ def _get_base_job_args(pipeline_args):
       str(pipeline_args.logging_interval_sec) + 's', '--zones'
   ] + pipeline_args.zones
   if pipeline_args.network:
-    job_args += ['--network', pipeline_args.network]
+    job_args.extend(['--network', pipeline_args.network])
   if pipeline_args.subnetwork:
-    job_args += ['--subnetwork', pipeline_args.subnetwork]
+    job_args.extend(['--subnetwork', pipeline_args.subnetwork])
   if pipeline_args.operation_label:
-    job_args += [
+    job_args.extend([
         '--labels', _DEEPVARIANT_LABEL_KEY + '=' + pipeline_args.operation_label
-    ]
+    ])
 
   return job_args
 
 
+def _generate_actions_for_make_example(
+    shard_start_index, shard_end_index, input_bam_file, is_gcsfuse_activated,
+    deep_variant_image, make_example_command_template):
+  """Returns a dictionary of actions for execution of make_examples stage.
+
+  Args:
+    shard_start_index: Index of first assigned shard to this worker (inclusive).
+    shard_end_index: Index of last assigned shard to this worker (inclusive).
+    input_bam_file: full path of bam file on gcs (gs://bucket/path/file.bam).
+    is_gcsfuse_activated: whether or not read input bam file using gcsfuse.
+    deep_variant_image: DeepVariant image given using --docker_image flag.
+    make_example_command_template: template command used in actions list.
+  """
+  gcs_bucket = _get_gcs_bucket(input_bam_file)
+  bam_file_relative_path = _get_gcs_relative_path(input_bam_file)
+
+  actions = []
+  if is_gcsfuse_activated:
+    io_command = 'mkdir -p'
+    for shard_index in range(shard_start_index, shard_end_index + 1):
+      local_dir = GCSFUSE_LOCAL_DIR_TEMPLATE.format(SHARD_INDEX=shard_index)
+      io_command += ' ' + local_dir
+      gcs_fuse_command = GCSFUSE_COMMAND_TEMPLATE.format(
+          BUCKET=gcs_bucket, LOCAL_DIR=local_dir)
+      actions.append({'imageUri': GCSFUSE_IMAGE,
```

Comment:
You could just create the directories before running fuse (so, eg, each action is now basically mkdir mount point; start gcsfuse on mount point